nemo/collections/llm/__init__.py

```diff
@@ -58,12 +63,16 @@
     Nemotron4Config340B,
     NemotronConfig,
     NemotronModel,
+    NVIDIAHybridConfig8b,
```

Comment:
Can we add "Mamba" to this name, otherwise might be confusing out of context

Reply:
changed to NVIDIAMambaHybridConfig8b